examples/jsm/loaders/AssimpLoader.js

```diff
@@ -878,7 +878,7 @@ AssimpLoader.prototype = {
 			this.a = 0;
 			this.toTHREE = function () {
 
-				return new Color( this.r, this.g, this.b, 1 );
+				return new Color( this.r, this.g, this.b );
```

Comment:
see https://github.com/mrdoob/three.js/pull/16572#discussion_r287752468

Reply:
BTW: This line has the same error: https://github.com/mrdoob/three.js/pull/16574/files#diff-0f9644de66075bc5c997fbc470225584R867

Reply:
yeah but there alpha isn't ensured to be equal to 1. So i didn't know how to solve that one yet

Reply:
I've edited the PR to remove both errors.